cache/ttl_cache_test.ts

```diff
@@ -126,17 +103,3 @@ Deno.test("TtlCache deletes entries", async (t) => {
     cache.set(1, "one");
   });
 });
-
-Deno.test("TTL implementation: setTimeout()", async (t) => {
-  await t.step(
-    "Schedules intervals less than 2 ** 31",
-    () => assertTimerDoesNotCompletePrematurely(2 ** 31 - 1),
-  );
-
-  await t.step(
-    "Fails to schedule intervals greater than or equal to 2 ** 31",
-    async () => {
-      await assertRejects(() => assertTimerDoesNotCompletePrematurely(2 ** 31));
-    },
-  );
-});
```

Comment:
@iuioiua I wasn't sure whether or not you wanted these V8-specific tests, but I thought it would be easier to revert their addition than to hold up review by lacking coverage.

Reply:
Yeah, I think it's fine to omit these tests.

Reply:
At least they're now documented as part of the context of this PR